projects/ngrx.io/content/marketing/resources.json

```
@@ -29,6 +29,12 @@
             "url":
               "https://www.youtube.com/playlist?list=PL8OUS498tQP3FFsZzULTGnbyIcILFjHd3",
             "rev": true
+          },
+          "International-Javascript-Conference-London-2019": {
```

Just a small tweak here, need these to be lowercase.

```suggestion
          "international-javascript-conference-london-2019": {
```